projects/view/client/rtProject/rtProjectInfoSection.js

```diff
@@ -0,0 +1,36 @@
+Template.rtProjectInfoSection.onRendered(function () {
```

Comment:
In this, and all, file(s), use two spaces per level of indentation. Convert tabs to spaces.